ts/screens/wallet/payment/ErrorTransactionScreen.tsx

```diff
@@ -0,0 +1,189 @@
+/**
+ * The screen to display to the user the various types of errors that occurred during the transaction.
+ * Inside the cancel and retry buttons are conditionally returned.
+ */
+import { Option } from "fp-ts/lib/Option";
+import { Button, Content, Text, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { NavigationInjectedProps } from "react-navigation";
+import { connect } from "react-redux";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import I18n from "../../../i18n";
+import { navigateToWalletHome } from "../../../store/actions/navigation";
+import { Dispatch } from "../../../store/actions/types";
+import {
+  paymentAttiva,
+  paymentIdPolling,
+  paymentVerifica
+} from "../../../store/actions/wallet/payment";
+import variables from "../../../theme/variables";
+import { PayloadForAction } from "../../../types/utils";
+
+type NavigationParams = {
+  error: Option<
+    PayloadForAction<
+      | typeof paymentVerifica["failure"]
+      | typeof paymentAttiva["failure"]
+      | typeof paymentIdPolling["failure"]
+    >
+  >;
+  onCancel: () => void;
+  onRetry?: () => void;
+};
+
+type OwnProps = NavigationInjectedProps<NavigationParams>;
+
+type Props = NavigationInjectedProps &
```

Comment:
You don't need NavigationInjectedProps again here.

Reply:
I solved in the commit 092ff8f